src/views/Forms.vue

```diff
@@ -37,38 +37,43 @@
                 </v-list-item>
               </v-list>
             </v-menu>
-            <!--          <v-btn-->
-            <!--            icon-->
-            <!--            @click="addForm(formType)">-->
-            <!--            <-->
-            <!--          </v-btn>-->
           </permission>
         </v-toolbar>
         <v-data-table
+          :sort-by.sync="sortBy"
           :headers="headers(formType)"
           hide-default-footer
-          :items="studyFormsByType(formType)">
-          <template v-slot:item="props">
-            <form-list-tile
-              :form="props.item.form"
-              :study-form="props.item"
-              :form-type="formType"
-              v-model="props.item.showHidden"
-              @save="updateForm"
-              @updateStudyForm="updateStudyForm"
-              @delete="deleteForm(props.item)"></form-list-tile>
-            <tr v-if="props.item.showHidden">
-              <td colspan="4">
-                <form-skips :form="props.item.form"></form-skips>
-              </td>
-            </tr>
+          :items="studyFormsByType[formType]">
+          :item-key="form.id"
+          <template v-slot:body="props">
+            <draggable
+              handle=".drag-handle"
+              :list="props.items"
+              @end="reorderForms"
+              tag="tbody">
+              <form-list-tile
+                v-for="item in props.items"
+                :key="item.id"
+                :form="item.form"
+                :study-form="item"
+                :form-type="formType"
+                v-model="item.showHidden"
+                @toggleFormSkips="toggleFormSkips"
+                @save="updateForm"
+                @updateStudyForm="updateStudyForm"
+                @delete="deleteForm(props.item)">
+              </form-list-tile>
+            </draggable>
           </template>
         </v-data-table>
       </v-card>
     </v-flex>
     <v-dialog v-model="showImportForm" @formImported="onFormImported(importedForm)" max-width="50em">
       <form-import :form-type="importFormType"></form-import>
     </v-dialog>
+    <v-dialog v-model="showFormSkips">
```

Comment:
Why not just use TrellisModal here?

Reply:
Ah, I see. We need a way to access the title slot from the modal as well.

Reply:
I think we can extend Trellis Modal to have an additional title slot, didn't seem worth the time for now. We can revisit.